docs/guides/javascript.rst

```diff
@@ -675,7 +675,21 @@ Note that WYSIWYG will be automatically attached to all instances of ``.elgg-inp
 .. code:: js
 
    require(['elgg/ckeditor'], function (editor) {
-       editor.bind('.my-text-area');
+      editor.bind('#my-text-area');
```

Comment:
Non-blocking suggestion: To me `editor` sounds like an editor instance. I guess I'd prefer something that sounds like an API/module. `ckeditor_api` maybe but it's not much better.